### ISSUE TITLE
3.2.47 - Picture is not updated automatically

### ISSUE THREAD

**Issue by edualm** [GeekMagicClock/smalltv-pro#96]
Hello,

After updating to 3.2.47, the picture is not updated automatically on the device after a new one is uploaded - it requires me to go back to the menu and then back to the Picture app for it to be updated.

This does not happen on the older version of the firmware (3.2.40) - once I delete and upload a new image, it's automatically shown on the Picture app.

**Comment by GeekMagicClock** [comment]
Yes, this is a bug in older version, the V3.3.47 correct this, you can enable the image auto display option on the web.

**Comment by edualm** [comment]
@GeekMagicClock It works fine in 3.2.40 though.

For reference, I use it to show custom information, I have a script that uploads a file to the TV every 5 minutes with updated data. Since we can't write our custom apps, that's the best I could think of!

**Comment by GeekMagicClock** [comment]
> @GeekMagicClock It works fine in 3.2.40 though.
> 
> For reference, I use it to show custom information, I have a script that uploads a file to the TV every 5 minutes with updated data. Since we can't write our custom apps, that's the best I could think of!

Upload a new file to the /image dir?

**Comment by edualm** [comment]
> Upload a new file to the /image **dir?**

That's what I'm doing. In `3.2.40` it works fine, in `3.2.47` I need to go back to the menu and then again into the Picture app for it to update.

**Comment by GeekMagicClock** [comment]
> > Upload a new file to the /image **dir?**
> 
> That's what I'm doing. In `3.2.40` it works fine, in `3.2.47` I need to go back to the menu and then again into the Picture app for it to update.

OK, let me check.

**Comment by Vladislaff** [comment]
Upload to image dir works for me, but /set?img=/image/xxxx.jpg retrns 200 with text FAIL.

**Comment by GeekMagicClock** [comment]
@edualm @Vladislaff  Hi the file list will be updated after the upload in v3.3.61 and after, please try here.  https://github.com/GeekMagicClock/smalltv-pro/issues/133#issuecomment-2841493581
